### ISSUE TITLE
Crashes on startup

### ISSUE THREAD

**Issue by joeky888** [yanus171/Handy-News-Reader#661]
Open the app and it crashes immediately.

LineageOS 17.1/Android 10 rooted

```yaml
UNCAUGHT EXCEPTION
-------------------
Stacktrace:
java.lang.NullPointerException: item must not be null
	at ru.yanus171.feedexfork.utils.FileUtils.createStorageList(FileUtils.kt:252)
	at ru.yanus171.feedexfork.service.FetcherService$1.run(FetcherService.java:305)
	at ru.yanus171.feedexfork.service.FetcherService.LongOper(FetcherService.java:608)
	at ru.yanus171.feedexfork.service.FetcherService.LongOper(FetcherService.java:597)
	at ru.yanus171.feedexfork.service.FetcherService.onHandleIntent(FetcherService.java:297)
	at android.app.IntentService$ServiceHandler.handleMessage(IntentService.java:78)
	at android.os.Handler.dispatchMessage(Handler.java:107)
	at android.os.Looper.loop(Looper.java:214)
	at android.os.HandlerThread.run(HandlerThread.java:67)

-------------------
Version : 0.15.15
Version Code: 225
Package : ru.yanus171.feedexfork
FilePath : /data/user/0/ru.yanus171.feedexfork/files
Phone Model: ASUS_X00TD
Android Version : 10
User : nguyenhung9xdev2020
Total Internal memory : 54350290944
Available Internal memory : 40637153280
-------------------
```

**Comment by KTHRANE** [comment]
Hi 

I see the same error after updating Oneplus 9 to Android 12

My Handy News version is: 0.16.7

Any workaround ?

**Comment by yanus171** [comment]
What error?


KTHRANE ***@***.***> 26 января 2022 г. 23:26:31 написал:

**Comment by KTHRANE** [comment]
Hi Yanus,

Sorry, I was commenting on an existing posting, that looked similar to 
mine - I later saw this was an old posting.

Below you can see the error I get after updating my Oneplus 9 to Android 
12. I get the same error whenever I try to start the app, reboot does 
not help ;-(. Do you want me to create a new entry in github ?

Thank you
Klaus

UNCAUGHT EXCEPTION
-------------------
Stacktrace:
java.lang.IllegalArgumentException: ru.yanus171.feedexfork: Targeting S+ 
(version 31 and above) requires that one of FLAG_IMMUTABLE or 
FLAG_MUTABLE be specified when creating a PendingIntent.
Strongly consider using FLAG_IMMUTABLE, only use FLAG_MUTABLE if some 
functionality depends on the PendingIntent being mutable, e.g. if it 
needs to be used with inline replies or bubbles.
at android.app.PendingIntent.checkFlags(PendingIntent.java:378)
at android.app.PendingIntent.getBroadcastAsUser(PendingIntent.java:648)
at android.app.PendingIntent.getBroadcast(PendingIntent.java:635)
at 
ru.yanus171.feedexfork.service.FetcherService.createCancelPI(FetcherService.java:2005)
at 
ru.yanus171.feedexfork.service.FetcherService.LongOper(FetcherService.java:608)
at 
ru.yanus171.feedexfork.service.FetcherService.LongOper(FetcherService.java:604)
at 
ru.yanus171.feedexfork.service.FetcherService.onHandleIntent(FetcherService.java:450)
at 
android.app.IntentService$ServiceHandler.handleMessage(IntentService.java:78)
at android.os.Handler.dispatchMessage(Handler.java:106)
at android.os.Looper.loopOnce(Looper.java:233)
at android.os.Looper.loop(Looper.java:344)
at android.os.HandlerThread.run(HandlerThread.java:67)

-------------------
Version : 0.16.7
Version Code: 239
Package : ru.yanus171.feedexfork
FilePath : /data/user/0/ru.yanus171.feedexfork/files
Phone Model: LE2113
Android Version : 12
User : root
Total Internal memory : 109636694016
Available Internal memory : 51244281856
-------------------

------ Original Message ------
